ci/jenkins/mellanox/scripts/start_ci.sh

```diff
@@ -94,10 +94,18 @@ EOF
     if [[ -z "${ANTREA_CNI_PR}${ANTREA_CNI_BRANCH}" ]];then
         change_image_name $ANTREA_CNI_HARBOR_IMAGE antrea/antrea-ubuntu
     fi
-    
-    if [[ -z "$(grep hw-offload $WORKSPACE/antrea-cni/build/yamls/antrea.yml)" ]];then
-        sed -i '/start_ovs/a\        - --hw-offload' $WORKSPACE/antrea-cni/build/yamls/antrea.yml
-    fi
+
+    change_image_name antrea/antrea-ubuntu antrea/antrea-ubuntu:ci
```

Comment:
why do we need to rename the image to `antrea/antrea-ubuntu:ci`?

Reply:
i though of changing it to a name that does not have an upstream image so that the CI will fail in case it could not find the image, but now that i think about it, we dont need that if we are going to change the antrea.yml anyway.